docs/hardware.md

```diff
@@ -0,0 +1,204 @@
+## Hardware Compatibility List (HCL)
+
+Devices listed on [Citrix Hypervisor's Hardware Compatibility List](http://hcl.xenserver.org/) are supported.
+
+For other hardware, see "Hardware outside the official HCL" below.
+
+## Hardware outside the official HCL
```

Comment:
This title is a bit long. "Unlisted hardware" might be shorted and better.

Reply:
Done